tutorials/transformations_02_coordinate_system_manager.ipynb

```diff
@@ -9,18 +9,6 @@
     "## Imports"
```

Comment:
<p>not only internally, but the logical relation of the systems is a tree. Eventually you can elaborate on that (1-2 sentences).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/BAMWelDX/weldx/pull/767/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>Think the rendered images should be enough, don't you think?</p>